skyportal/tests/frontend/test_gcnevents.py

```diff
@@ -85,3 +91,123 @@ def test_gcnevents(
     driver.wait_for_xpath('//*[text()="180116 00:36:53"]')
     driver.wait_for_xpath('//*[text()="Fermi"]')
     driver.wait_for_xpath('//*[text()="GRB"]')
+
+    # test modify sources form
+    driver.wait_for_xpath('//*[@id="root_startDate"]').send_keys('01/15/2018')
+    driver.wait_for_xpath('//*[@id="root_startDate"]').send_keys(Keys.TAB)
+    driver.wait_for_xpath('//*[@id="root_startDate"]').send_keys('01:01')
+    driver.wait_for_xpath('//*[@id="root_startDate"]').send_keys('P')
+    driver.wait_for_xpath('//*[@id="root_endDate"]').send_keys('01/18/2018')
+    driver.wait_for_xpath('//*[@id="root_endDate"]').send_keys(Keys.TAB)
+    driver.wait_for_xpath('//*[@id="root_endDate"]').send_keys('01:01')
+    driver.wait_for_xpath('//*[@id="root_endDate"]').send_keys('P')
+    driver.wait_for_xpath('//*[@id="root_localizationName"]')
+    driver.click_xpath('//*[@id="root_localizationName"]')
+    driver.wait_for_xpath('//li[contains(text(), "214.74000_28.14000_11.19000")]')
+    driver.click_xpath('//li[contains(text(), "214.74000_28.14000_11.19000")]')
+
+    submit_button_xpath = '//button[@type="submit"]'
+    driver.wait_for_xpath(submit_button_xpath)
+    driver.click_xpath(submit_button_xpath)
+
+    # check for object
+    driver.wait_for_xpath(f'//*[text()[contains(.,"{obj_id}")]]')
+
+
+# @pytest.mark.flaky(reruns=2)
+def test_gcnevents_observations(
+    driver, user, super_admin_token, upload_data_token, view_only_token, ztf_camera
+):
+
+    datafile = f'{os.path.dirname(__file__)}/../data/GW190425_initial.xml'
+    with open(datafile, 'rb') as fid:
+        payload = fid.read()
+    data = {'xml': payload}
+
+    status, data = api('POST', 'gcn_event', data=data, token=super_admin_token)
+    assert status == 200
+    assert data['status'] == 'success'
+
+    name = str(uuid.uuid4())
```

Comment:
Name of what is this? There's a bunch of names saved it would be easier to read if you say what this names.

Reply:
great point.